ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3405,6 +3405,64 @@ AHU_ECON_EFSS2X_EFVSC2X_EPC_HT4SC_HTWHLSTC_SFC_SFSS2X_SFVSC2X_SSPC:
   - SFVSC2X
   - SSPC
 
+AHU_DFSS_DFVSC_DX2DC_ECOND_EFSS_EFVSC_HWDC_ZHM:
+  description: "Single zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DFVSC
+  - DX2DC
+  - ECOND
+  - EFSS
+  - EFVSC
+  - HWDC
+  - ZHM
+
+AHU_BSPC_DX2SC_ECON_EFSS_FDPM_SARC_SFSS_SFVSC_SSPC:
+  description: "Two stage compressor AHU with differential pressure filter monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - BSPC
+  - DX2SC
+  - ECON
+  - EFSS
+  - FDPM
+  - SARC
+  - SFSS
+  - SFVSC
+  - SSPC
+
+AHU_DFSS_DFVSC_DSP_DX4DC_ECONZ_EFSS_ZSPC:
+  description: "Single zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DFVSC
+  - DSP
+  - DX4DC
+  - ECONZ
+  - EFSS
+  - ZSPC
+
+AHU_BSPC_DX4SC_EC2SC_ECON_EFSS2X_EFVSC2X_SARC_SFSS4X_SFVSC_SSPC:
+  description: "Multi-zone AHU."
```

Comment:
Add comment about evaporative precooling, which it looks like this does. Please also confirm that the `EC2SC` is for supply temp control, and that this isn't actually evaporative cooling for the condensing coils.

Reply:
It appears to be supply controlled. This is for US-MTV-1015 AC-1.